LanguageFeatures/Patterns/if_case_A02_t03.dart

```diff
@@ -0,0 +1,271 @@
+    Expect.setEquals({1, 3, 4}, testRelational(25, true));
+    Expect.setEquals({1, 2, 4}, testConstant(30, 42));
+    Expect.setEquals({1, 3, 4}, testConstant(30, 41));
+    Expect.setEquals({1, 3, 4}, testConstant(31, 42));
+    Expect.setEquals({1, 2, 4}, testParenthesized1(40, 40));
+    Expect.setEquals({1, 3, 4}, testParenthesized1(42, 40));
+    Expect.setEquals({1, 3, 4}, testParenthesized1(40, 42));
+    Expect.setEquals({1, 2, 4}, testParenthesized2(42));
+    Expect.setEquals({1, 3, 4}, testParenthesized2(40));
+    Expect.setEquals({1, 2, 4}, testCast(42));
+    Expect.setEquals({1, 3, 4}, testCast(41));
+    Expect.setEquals({1, 2, 4}, testNullCheck1(1));
+    Expect.setEquals({1, 3, 4}, testNullCheck1(-1));
+    Expect.setEquals({1, 3, 4}, testNullCheck1(null));
+    Expect.setEquals({1, 2, 4}, testNullAssert(1));
+    Expect.setEquals({1, 3, 4}, testNullAssert(-1));
+    Expect.throws(() {testNullAssert(null);});
+    Expect.setEquals({1, 2, 4}, testVariable(1));
+    Expect.setEquals({1, 3, 4}, testVariable(-1));
+    Expect.setEquals({1, 2, 4}, testList1([1, 42]));
+    Expect.setEquals({1, 3, 4}, testList1([2, 42]));
+    Expect.setEquals({1, 3, 4}, testList1([1, 0]));
+    Expect.setEquals({1, 3, 4}, testList1([1, 42, 0]));
+    Expect.setEquals({1, 2, 4}, testList2([1, -42]));
+    Expect.setEquals({1, 3, 4}, testList2([2, -42]));
+    Expect.setEquals({1, 3, 4}, testList2([1, 0]));
+    Expect.setEquals({1, 2, 4}, testList2([1, -42, 0]));
+    Expect.setEquals({1, 2, 4}, testList3([1, 2, 3]));
+    Expect.setEquals({1, 3, 4}, testList3([2, 2]));
+    Expect.setEquals({1, 3, 4}, testList3([1, 0]));
+    Expect.setEquals({1, 2, 4}, testList3([1, -42, 0]));
+    Expect.setEquals({1, 2, 4}, testMap1({"key1": 1, "key2": 2}));
+    Expect.setEquals({1, 3, 4}, testMap1({"key1": 2, "key2": 2}));
+    Expect.setEquals({1, 3, 4}, testMap1({"key1": 1, "key2": -2}));
+    Expect.setEquals({1, 3, 4}, testMap1({"key1": 1, "key2": 2, "key3": 3}));
+    Expect.setEquals({1, 2, 4}, testMap2({"key1": 1, "key2": -2}));
+    Expect.setEquals({1, 3, 4}, testMap2({"key1": 2, "key2": -2}));
+    Expect.setEquals({1, 3, 4}, testMap2({"key1": 1, "key2": 2}));
+    Expect.setEquals({1, 2, 4}, testMap2({"key1": 1, "key2": -2, "key3": 3}));
+    Expect.setEquals({1, 2, 4}, testRecord1((1, 2)));
+    Expect.setEquals({1, 3, 4}, testRecord1((2, 2)));
+    Expect.setEquals({1, 3, 4}, testRecord1((1, -2)));
+    Expect.setEquals({1, 2, 4}, testRecord2((1, 2, n: 1)));
+    Expect.setEquals({1, 3, 4}, testRecord2((2, 2, n: 1)));
+    Expect.setEquals({1, 3, 4}, testRecord2((1, 2, n: -1)));
+    Expect.setEquals({1, 3, 4}, testRecord2((1, n: 1)));
+    Expect.setEquals({1, 3, 4}, testRecord2((1, 2, n: 1, n2: 2)));
+    Expect.setEquals({1, 2, 4}, testObject1(Square(3)));
+    Expect.setEquals({1, 3, 4}, testObject1(Circle(3)));
+    Expect.setEquals({1, 3, 4}, testObject1(Square(1)));
+    Expect.setEquals({1, 2, 4}, testObject1(Circle(3)));
```

Comment:
```suggestion
    Expect.setEquals({1, 2, 4}, testObject2(Circle(3)));
```
Same corrections again in the following lines.